scripts/change-log-generator.js

```diff
@@ -106,21 +88,35 @@ function getPreviousReleaseBranch(releaseBranch) {
 }
 
 function validateReleaseBranch(releaseBranch) {
-  if (!(releaseBranch && RELEASE_REGEX.exec(releaseBranch))) {
+  if (!(releaseBranch && constants.RELEASE_REGEX.exec(releaseBranch))) {
     console.log(
       "Invalid release '" + releaseBranch + "'. Expected format [xx.yy.z]."
     );
     process.exit(-1);
   }
 }
 
+/**
+ * Create the changelog branch for committing these changes.
+ * If the branch already exists, check it out to append any
+ * new changes.
+ */
 function getNewChangeLogBranch(releaseBranch) {
   if (ADD_VERBOSE_LOGGING) {
     console.log('\nStep 2: Create new change log branch.');
   }
   var changeLogBranch =
-    CHANGE_LOG_BRANCH + releaseBranch.replace('origin/release/v', '');
-  shell.exec(util.format(GIT_CHECKOUT, changeLogBranch, releaseBranch));
+    constants.CHANGE_LOG_BRANCH +
+    releaseBranch.replace(constants.RELEASE_BRANCH_PREFIX, '');
+  shell.exec(
+    `git checkout $(git show-ref --verify --quiet refs/heads/${changeLogBranch} || echo '-b') ${changeLogBranch} ${releaseBranch}`
```

Comment:
Undoing this piece - this would allow the logic to still check out the change log branch if it was already generated. I went back and forth on adding this in, but have decided this might not be the time to have this in yet.